api/app/src/main/kotlin/packit/exceptions/PackitExceptionHandler.kt

```diff
@@ -0,0 +1,50 @@
+package packit.exceptions
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.servlet.NoHandlerFoundException
+import packit.model.ErrorDetail
+import java.util.*
+
+@ControllerAdvice
+class PackitExceptionHandler
+{
+    @ExceptionHandler(NoHandlerFoundException::class)
+    fun handleNoHandlerFoundException(error: Exception): Any
+    {
+        return handleErrorPage(error)
+    }
+
+    @ExceptionHandler(PackitException::class)
+    fun handlePackitException(
+        error: PackitException
+    ): ResponseEntity<String>
+    {
+        val resourceBundle = getBundle()
+
+        return ErrorDetail(error.httpStatus, resourceBundle.getString(error.key))
+            .toResponseEntity()
+    }
+
+    private fun handleErrorPage(e: Exception): ResponseEntity<String>
```

Comment:
this doesn't return a page and only gets called once, suggest we remove it altogether and just move line 33 to line 17?